src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -1,408 +0,0 @@
-                    return DiagnosticAnalysisResult.CreateEmpty(project.Id, VersionStamp.Default);
-
-                return builder.ToResult();
-            }
-
-            private void AddToInMemoryStorage(
-                VersionStamp serializerVersion, ProjectOrDocumentId key, string stateKey, ImmutableArray<DiagnosticData> diagnostics)
-            {
-                InMemoryStorage.Cache(_owner.Analyzer, (key, stateKey), new CacheEntry(serializerVersion, diagnostics));
-            }
-
-            private bool TryGetDiagnosticsFromInMemoryStorage(VersionStamp serializerVersion, TextDocument document, Builder builder)
-            {
-                var success = true;
-                var project = document.Project;
-                var documentId = document.Id;
-
-                var diagnostics = GetDiagnosticsFromInMemoryStorage(serializerVersion, new(documentId), SyntaxStateName);
-                if (!diagnostics.IsDefault)
-                {
-                    builder.AddSyntaxLocals(documentId, diagnostics);
-                }
-                else
-                {
-                    success = false;
-                }
-
-                diagnostics = GetDiagnosticsFromInMemoryStorage(serializerVersion, new(documentId), SemanticStateName);
-                if (!diagnostics.IsDefault)
-                {
-                    builder.AddSemanticLocals(documentId, diagnostics);
-                }
-                else
-                {
-                    success = false;
-                }
-
-                diagnostics = GetDiagnosticsFromInMemoryStorage(serializerVersion, new(documentId), NonLocalStateName);
-                if (!diagnostics.IsDefault)
-                {
-                    builder.AddNonLocals(documentId, diagnostics);
-                }
-                else
-                {
-                    success = false;
-                }
-
-                return success;
-            }
-
-            private bool TryGetProjectDiagnosticsFromInMemoryStorage(VersionStamp serializerVersion, Project project, Builder builder)
-            {
-                var diagnostics = GetDiagnosticsFromInMemoryStorage(serializerVersion, new(project.Id), NonLocalStateName);
-                if (!diagnostics.IsDefault)
-                {
-                    builder.AddOthers(diagnostics);
-                    return true;
-                }
-
-                return false;
-            }
-
-            private ImmutableArray<DiagnosticData> GetDiagnosticsFromInMemoryStorage(
-                VersionStamp serializerVersion, ProjectOrDocumentId key, string stateKey)
-            {
-                return InMemoryStorage.TryGetValue(_owner.Analyzer, (key, stateKey), out var entry) && serializerVersion == entry.Version
-                    ? entry.Diagnostics
-                    : default;
-            }
-
-            private void RemoveInMemoryCache(DiagnosticAnalysisResult lastResult)
```

Comment:
all the saving code was removed (since we now save into a CWT).  so the 'in memory cache' is always empty.  so all the removal code is unnecessary and can be removed.